crates/nomination/src/lib.rs

```diff
@@ -87,6 +87,9 @@ decl_storage! {
 
         /// Map of Vault Operators
         Operators: map hasher(blake2_128_concat) T::AccountId => Operator<T::AccountId, T::BlockNumber, Backing<T>>;
+
+        // map from vault_id to to nominator_id
+        Nominators: map hasher(blake2_128_concat) T::AccountId => T::AccountId;
```

Comment:
This map was placed in the vault_registry so direct operator withdrawals can be disabled. Since that check has been removed, I'm not sure what's stopping operators from calling the `withdraw` dispatchable function in the vault registry

Reply:
I want to move back to using that for vault withdrawals (regardless if it is an operator). With the new slashing changes it will only be able to withdraw it's own collateral anyway - we can add de-bond period later.